packages/gatsby-image/src/__tests__/index.js

```diff
@@ -56,8 +124,18 @@ describe(`<Image />`, () => {
     expect(component).toMatchSnapshot()
   })
 
+  it(`should render multiple fixed image variants`, () => {
+    const component = setupImages()
+    expect(component).toMatchSnapshot()
+  })
+
+  it(`should render multiple fluid image variants`, () => {
+    const component = setupImages(true)
+    expect(component).toMatchSnapshot()
+  })
+
   it(`should have correct src, title, alt, and crossOrigin attributes`, () => {
-    const imageTag = setup().querySelector(`picture img`)
+    const imageTag = setup().querySelectorAll(`picture img`)[1]
```

Comment:
This was necessary since now there are two `<picture>` tags for images with base64 or tracedSVG placeholders. Feels a little janky to me, and I'd be open to ideas on refactoring.

Reply:
At the very least the `1` should not be used directly, assign it to a const that identifies what the index is intended to represent.